### ISSUE TITLE
Can I make a release without having my current commit  `git tag`-ed ?

### ISSUE THREAD

**Issue by ananya7** [travis-ci/travis-ci#8749]
Say, I have two projects being built by travis.

Repository Name  | Repository Visibility | Repo Tag
---                          |     ---                        |  ---
backend-buster     | private                     | v0.1 + some 10-20 commits
frontend-fun | public                       | v0.2

Lets say I released v0.1 of my `backend-buster` (private repository) which is used by `frontend-fun` application currently at `v0.2` in a public repository.

I am now heading towards v0.2 of my backend logic(private repo) and I need a working binary of my application to be used in another application(public repo)which is in `v0.2`. 
How can travis get the binary from the backend-buster's releases tab while building `frontend-fun` ?

**Comment by ananya7** [comment]
I have the .travis.yml 

```
deploy:                                                                                                                    
  provider: releases   # specify that it has to deploy the binaries in Github Releases                                               skip_cleanup: true    #Otherwise Travis Cleans all the files created during build                           
  overwrite: true        #should overwrite the release binary, or maybe overwrite the code.                                                                                                  
  api_key:                                                                                                                 
    secure: <redacted>
  file_glob: true                                                                                                  
  file: dist/logicpro-*.tar
    on:                                                                                                                      
      repo: applepie/banana                                                                                                  
      branch: master
```
Setting `overwrite: true` doesn't upload the binaries with every commit to the tagged release.

**Comment by BanzaiMan** [comment]
I think this is an effective duplicate of https://github.com/travis-ci/travis-ci/issues/8185.

TL; DR You don't need a tag to have a GitHub Release.
